stdlib/public/core/ArrayCast.swift

```diff
@@ -60,7 +59,7 @@ internal struct _UnwrappingFailed : Error {
 }
 
 extension Optional {
-  @inlinable // FIXME(sil-serialize-all)
+  @usableFromInline
   internal func unwrappedOrError() throws -> Wrapped {
     if let x = self { return x }
     throw _UnwrappingFailed()
```

Comment:
This is clever but just writing the loop out long-hand is probably better regardless.

Reply:
Agreed!